src/NHibernate/Tool/hbm2ddl/SchemaUpdate.cs

```diff
@@ -141,7 +141,8 @@ public void Execute(Action<string> scriptAction, bool doUpdate)
 			autoKeyWordsImport = autoKeyWordsImport.ToLowerInvariant();
 			if (autoKeyWordsImport == Hbm2DDLKeyWords.AutoQuote)
 			{
-				SchemaMetadataUpdater.QuoteTableAndColumns(configuration);
+				SchemaMetadataUpdater.Update(configuration, dialect);
```

Comment:
For consistency with `SchemaExport`, it could be done on a `Initialize` method, called only once from method requiring it. `SchemaUpdate` can be used by users, we cannot assume they will never call many time its execute.